src/notebook.c

```diff
@@ -473,6 +496,12 @@ static void show_tab_bar_popup_menu(GdkEventButton *event, GeanyDocument *doc)
 	g_signal_connect(menu_item, "activate", G_CALLBACK(on_close_other_documents1_activate), doc);
 	gtk_widget_set_sensitive(GTK_WIDGET(menu_item), (doc != NULL));
 
+	menu_item = ui_image_menu_item_new(GTK_STOCK_CLOSE, _("Close Documents to the _Right"));
```

Comment:
Yeah. Is there a way with gettext to transmit a note with the string, or just a normal comment? Also, does GtkNotebook handle the backwards indexing already when in RTL mode?